src/extras/MovieClip.js

```diff
@@ -56,13 +56,13 @@ function MovieClip(textures)
     this.onComplete = null;
 
     /**
-     * The MovieClips current frame index (this may not have to be a whole number)
+     * Elapsed time since animation has been started, used internally to display current texture
```

Comment:
If it is used internally, want to just make it internal? e.g. _currentTime, mark with @private